dev-tools/mage/gotest.go

```diff
@@ -126,14 +125,14 @@ func DefaultTestBinaryArgs() TestBinaryArgs {
 // Use MODULE=module to run only tests for `module`.
 func GoTestIntegrationForModule(ctx context.Context) error {
 	module := EnvOr("MODULE", "")
-	modulesFileInfo, err := ioutil.ReadDir("./module")
+	modulesDirEntry, err := os.ReadDir("./module")
```

Comment:
Conversion to os.ReadDir, the return type changed to fs.DirEntry. Double check this code.